public/manifest.json

```diff
@@ -1,25 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
+    "short_name": "Backdoor",
+    "name": "Backdoor to Frigidsec",
```

Comment:
Change this to just "Backdoor" or something like "Backdoor - Cybersecurity forum"